test/http-mock.ts

```diff
@@ -77,30 +106,147 @@ export function scope(basePath: BasePath, options?: nock.Options): nock.Scope {
         result.body = requestBody;
       }
     }
-    requestLog.push(result);
+    requestsDone.push(result);
   });
 }
 
-export function getTrace(): RequestLogItem[] /* istanbul ignore next */ {
-  const errorLines: string[] = [];
-  if (missingLog.length) {
-    errorLines.push('Missing mocks:');
-    errorLines.push(...missingLog);
+export function getTrace(): RequestLog[] {
+  return requestsDone;
+}
+
+function massageHttpMockStacktrace(err: Error): void {
+  if (!err.stack) {
+    return;
   }
-  if (!nock.isDone()) {
-    errorLines.push('Unused mocks:');
-    errorLines.push(...nock.pendingMocks().map((x) => `  ${x}`));
+
+  const state = expect.getState();
+  if (!state.currentTestName || !state.testPath) {
+    return;
+  }
+
+  const content = fs.readFileSync(state.testPath, { encoding: 'utf8' });
```

Comment:
this fails if fs-extra is mocked inside test

Reply:
Good catch